<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>抖音视频下载器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,1,0,0,0">
      <item>
       <widget class="QLabel" name="URL">
        <property name="text">
         <string>链接地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="url_input"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_resolution">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>解析</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_favorites">
        <property name="text">
         <string>我的收藏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_likes">
        <property name="text">
         <string>我的喜欢</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_login">
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTableWidget" name="table_widget">
        <property name="maximumSize">
         <size>
          <width>2000</width>
          <height>500</height>
         </size>
        </property>
        <property name="rowCount">
         <number>1</number>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>95</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <row/>
        <column>
         <property name="text">
          <string>选择</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>序号</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>保存位置：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="save_directory"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_select_file">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_download">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>下载</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
